.github/workflows/ci.yaml

```diff
@@ -110,61 +204,76 @@ jobs:
         toolchain: ${{ env.rust_version }}
         components: ${{ env.rust_toolchain_components }}
         default: true
-    - name: Format Check
-      run: rustfmt --check --edition 2018 $(find -name '*.rs' -print | grep -v /target/)
-      # windows doesn't support using --check like this
-      if: ${{ matrix.os == 'ubuntu-latest' }}
-    - name: clippy check
-      run: cargo clippy --all-features -- -D warnings
-      working-directory: ${{ matrix.runtime }}
-      # don't bother running Clippy twice, it will have the same results on Windows
-      if: ${{ matrix.os == 'ubuntu-latest' }}
-    - name: run tests
-      run: cargo test --all-features
-      working-directory: ${{ matrix.runtime }}
-    - name: generate docs
-      run: cargo doc --no-deps --document-private-items --all-features
-      working-directory: ${{ matrix.runtime }}
-
-  # Psuedo-job that depends on the rust-tests job so that we don't have to enter
-  # the myriad of test matrix combinations into GitHub's protected branch rules
-  require-rust-tests:
-    needs: rust-tests
-    # Run this job even if its dependency jobs fail
-    if: always()
-    runs-on: ubuntu-latest
-    name: Rust Tests Matrix Success
-    steps:
-    - name: Verify jobs succeeded
-      uses: re-actors/alls-green@3a2de129f0713010a71314c74e33c0e3ef90e696
-      with:
-        jobs: ${{ toJSON(needs) }}
+    - name: Run tests
+      shell: bash
+      run: |
+        for runtime_path in "rust-runtime" "aws/rust-runtime"; do
+          pushd "${runtime_path}" &>/dev/null
+          cargo test --all-features
+          cargo doc --no-deps --document-private-items --all-features
+          popd &>/dev/null
+        done
```

Comment:
It's similar to, but not the same as, the `check-rust-runtimes-and-tools` script. It doesn't run clippy on the Rust runtimes though, and it also doesn't check the tools since we don't use the tools on Windows.